index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 4: Replication Project</title>
+  <title>Week 4: Replication Project (Brad Matheson)</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .section {
```

Comment:
Wouldn't this be nicer with a few different colors?
```css
    .point {
      stroke-width: 1;
      opacity: .5;
    }

    .D {
      stroke: rgb(51, 122, 183);
      fill: rgba(51, 122, 183, .5);
    }

    .R {
      fill: rgba(240, 72, 51, .5);
      stroke: rgb(240, 72, 51);
    }
```
And then you can assign these new colored classes with .classed:
```js
    points.enter().append('circle')
      .classed('point', true)
      .classed('R', function(d){
        if (d.party == "rep") { return true }
      })
      .classed('D', function(d){
        if (d.party == "dem") { return true }
      })
      .attr('r', 5);
```

Note you _can_ assign multiple classes with .attr, but as you can see below, it's based on string concatenation and can get really messy. The underlying aspect that makes this tricky is that .attr('class', 'whatever') will replace the current class with 'whatever', not append it on, just making .classed() preferable.
```js
    points.enter().append('circle')
      .attr('class', function(d){
        if (d.party == "rep") {return 'circle R' }
        else if (d.party == "dem") {return 'circle D' }
      })
      .attr('r', 5);
```